### DISCUSSION TITLE
Instructions to disable console setting set by the  "Enable console" item listed under "Advanced settings"

### DISCUSSION THREAD

**Discussion by ovalhed** [valinet/ExplorerPatcher#3345] [Q&A] [answered]
While updating options in the Properties screen, tab  "Advanced" I mistakenly clicked on the "Enable console" link and now have open Explorer screen running on my laptop. I did look in the "About advanced settings" and "Enable console"  after not finding a "Disable console" link in any tab in the program properties.
Computer is a Lenovo T590 running Windows 11 Pro.
I am not a programmer or coder, so I haven't a clue where in the registry the "Enable console" feature was enabled.
No luck finding the term "console" , "Explorer console" or "Terminal console" in Services editor.
All I have found is the last sentence in the "About advanced settings" I'm quoting here:
     > To be able to close it, disable "Enable console", then close the window.
I suggest disable console instructions be added to the enable console discussion in the About Advanced Settings for future unskilled persons such as myself can fix the self-created problem and not pester the programmers and coders here.

Thank you in advance for your help!

**Comment by pyrates999** [comment, marked as answer]
Just untick Enable Console and restart file explorer.

**Comment by ovalhed** [comment]
Thank you for the quick response and simple instruction that I overlooked trying yesterday.
I did untick Enable Console, restarted file explorer and voila, good as new.
